### ISSUE TITLE
Display Hi-Res Images

### ISSUE THREAD

**Issue by RachelDCE** [curationexperts-deprecated/tenejo-v1#66]
**ISSUE**
When I try to display a really big image, RIIIF crashes

**ACCEPTANCE**
- [x] Tenejo does not crash when panning or zooming really large images

There are sample images on the server in the `/opt/uploads/big-files/` folder - especially `Mississippi 4th dist 1.tif` and `Mississippi Pano 2.tif`

Child of #37

**Comment by bess** [comment]
We now have the ability to run our image processing against cantaloupe, externally. However, we still need to decide:
1) Do we want to run a separate cantaloupe server for each tenejo instance? Or do we want to set them up to run on the same box? 
2) Do we want to set expectations by giving a max image size that the system can handle? 

Currently, tenejo.curationexperts.com is set to run against an 8CPU 32G cantaloupe box. 
The system will not *crash* if given a huge TIFF, but it also will not perform as we'd like. I ran this test with a 207Mb file: https://tenejo.curationexperts.com/concern/works/k0698749q

It never serves out in a performant way*, and it uses a huge amount of system resources. If we want to support very large image files like this we should investigate JPEG2000 image compression. It may be that serving out large tiff files without background transformation isn't a good option.

* Actually, I came back the next day and it sort of looks okay. Maybe cantaloupe just needs some time to catch up and create derivatives? 

![big_image_2.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5515e9ab52d65b2000638516/125516c66e82c728ace21e0d46ea9d9d1c8b8ea888ec8011e611b2843b492d762a10e17132d0e179e61622575e5f47fa460a5844fcf8da2bf4b06e228c4e17ff933c0120996048e6edc00db513bb27814e80b2b48b45c0e8382ce38d8c5d73.png)
